Ska/DBI/DBI.py

```diff
@@ -170,7 +174,7 @@ def fetch(self, expr, vals=None,):
 
         :rtype: Generator that will get one row of database as dict() via next()
         """
-        self.execute(expr, vals)
+        self.execute(expr, vals, commit=False)
```

Comment:
This change will apply to sqlite as well.  I can code around it (make conditional for sybase), but I think it is ok?  For sybase, if the "select" statement is "committed", the cursor has nothing left to do and apparenetly cannot return the results.

Reply:
I have no idea if it's OK in sqlite, but I guess if tests and ska_testr pass then we're good.

Reply:
Thanks.  The Python sqlite docs suggest to me that this is a no-op difference on that side https://docs.python.org/3/library/sqlite3.html#controlling-transactions , and that if we really care we probably need to add `isolation_level` control to the `DBI` init.